Dockerfile

```
@@ -67,6 +67,40 @@ RUN cd /tmp/prototool && \
 
 RUN upx --lzma /usr/local/bin/*
 
+FROM swift:5.1 AS swift-builder
+ENV SWIFT_PROTO_VERSION=1.7.0
+RUN set -ex; \
+  mkdir -p /tmp/swift-proto; \
+  git clone -c advice.detachedHead=false --depth 1 -b ${SWIFT_PROTO_VERSION} \
+    https://github.com/apple/swift-protobuf.git /tmp/swift-proto; \
+  cd /tmp/swift-proto; \
+  swift build -c release; \
```

If you compile the plugin with a static stdlib you should not need to copy the other files.

```suggestion
  swift build --static-swift-stdlib  -c release; \
```

---

It does remove a couple of libraries at the expense of binary size. I have added the switch nevertheless. I have also automated the file copying to make this less brittle